addons/mass_mailing/models/mass_mailing.py

```diff
@@ -809,6 +812,8 @@ def convert_link(self, cr, uid, ids, context=None):
             res[mass_mailing.id] = mass_mailing.body_html if mass_mailing.body_html else ''
             
             for match in re.findall(URL_REGEX, res[mass_mailing.id]):
+                if '/unsubscribe_from_list' in match[1]:
```

Comment:
If this link is also converted into shortened URL, there won't be any way to find it in mail body and so we won't be able to replace the link at appropriate place while sending mail.